contracts/atm/TLRToken.sol

```diff
@@ -102,6 +102,8 @@ contract TLRToken is
         address atm
     ) external initializer() isNotInitialized() {
         require(cap > 0, "CAP_CANNOT_BE_ZERO");
+        require(atm != address(0x0), "ATM_CANNOT_BE_ZERO_ADDRESS");
```

Comment:
Probably we should use the same .isContract() validation for the ATM address, this would check both if not 0x0 and is a contract address.

Reply:
I agree

Reply:
Changed